packages/block-library/src/media-text/index.js

```diff
@@ -66,6 +65,13 @@ const blockAttributes = {
 		type: 'boolean',
 		default: false,
 	},
+	imageFill: {
```

Comment:
We need to reset these two attributes when the user selects a video as the media element, after having first selected an image. Currently, we have a bug if the user selects an image and then changes to a video the focal point picker continues to appear (without any image there).

Related question: Would the fill mode also make sense for videos? Should it just be called fill to allow a future addition?

Reply:
Good catch, added a reset for the two attributes!